src/sinks/gcp/cloud_storage.rs

```diff
@@ -145,8 +146,8 @@ impl Encoding {
 #[derivative(Default)]
 enum Compression {
     #[derivative(Default)]
-    Gzip,
     None,
+    Gzip,
```

Comment:
Why does this change the default?

Reply:
I though if config now will have default option as `none` it would be good have default variant `none` too.
Should I change it back?

Reply:
I would not change the default in this PR. If you think it should be changed, handle that separately.